### PR TITLE
update-1.0.7

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: qp_shogun/shogun/shogun.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def generate_shogun_assign_taxonomy_commands(temp_dir, parameters):
     ext = aln2ext[parameters['aligner']]
     output_fp = join(temp_dir, 'profile.tsv')
     cmds.append(
-        'shogun assign-taxonomy '
+        'shogun assign_taxonomy '
         '--aligner {aligner} '
         '--database {database} '
         '--input {input} --output {output}'.format(
```
Comment (qiyunzhu) [inline qp_shogun/shogun/shogun.py:87]: Would be good to add `--percent_id 0.95` to the command parameters for the sake of shotgun metagenomics (SHOGUN default 0.98 is for amplicon).

**Review by qiyunzhu — COMMENTED** [review]
@antgonza Looks all good to me! I made a minor comment. Up to you whether you want to change now. It does not have to be hard-coded but can be a user-selectable option.

File: qp_shogun/shogun/tests/test_shogun.py [hunk 2]
```diff
@@ -48,7 +49,8 @@ def setUp(self):
         self.params = {
             'Database': join(self.db_path, 'rep82'),
             'Aligner tool': 'bowtie2',
-            'Number of threads': 5
+            'Number of threads': 5,
+            'Percent ID': '0.95',
```
Comment (qiyunzhu) [inline qp_shogun/shogun/tests/test_shogun.py]: Minor comment: this "ID" is "identity" not "identifier".

Comment (antgonza) [reply qp_shogun/shogun/tests/test_shogun.py]: that is correct, do you have a suggestion for the name or is it fine to leave as is?

Comment (qiyunzhu) [reply qp_shogun/shogun/tests/test_shogun.py]: How about just "Percent identity"?

File: .travis.yml [hunk 3]
```diff
@@ -1,64 +1,67 @@
...
+  - wget https://github.com/knights-lab/BURST/releases/download/v0.99.8/burst_linux_DB15
   - chmod 755 burst_linux_DB15
   - mv burst_linux_DB15 $CONDA_PREFIX/bin/burst15
+  # making sure we have the correct software versions
+  - shogun_version=`shogun --version`
+  - bowtie2_version=`bowtie2 --version`
+  - burst15_version=`burst15 -h | grep BURST`
+  - utree_version=`(utree-search_gg || true)`
+  - if [[ $shogun_version != *"1.0.7"* ]]; then echo "wrong shogun version", $shogun_version; exit 1; fi
+  - if [[ $bowtie2_version != *"2.3.5"* ]]; then echo "wrong bowtie2 version", $bowtie2_version; exit 1; fi
```
Comment (qiyunzhu) [inline .travis.yml:54]: Just FYI: Bowtie2 is backward- / forward- compatible. Either 2.3.2 or 2.3.6 (in the future) should just work. Whereas the versions for BURST and UTree are very strict.

**Review by qiyunzhu — APPROVED** [review]

**Pull request merged**